src/Shared/SFA.DAS.SharedOuterApi.UnitTests/InnerApi/Requests/WhenBuildingGetAggregatedEmployerRequestsRequest.cs

```diff
@@ -0,0 +1,20 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.ProviderRequestApprenticeTraining.InnerApi.Requests;
+
+namespace SFA.DAS.SharedOuterApi.UnitTests.InnerApi.Requests
+{
+    public class WhenBuildingGetAggregatedEmployerRequestsRequest
```

Comment:
This test should be in the SFA.DAS.ProviderRequestApprenticeTraining project

Reply:
I've moved it to the correct project and also added a test for the corresponding response